### DISCUSSION TITLE
sheilds.io test

### DISCUSSION THREAD

**Discussion by aaronparker** [TimMangan/App-Info#4] [Ideas]
@TimMangan I created test page here using sheilds.io badges for statuses: https://github.com/TimMangan/App-Info/tree/shields-test/docs/Windows/Adobe/AdobeReader%20DC. A simple test as I was doing this for something else. I think the result looks good.

Here's a sample:

![badge](https://img.shields.io/badge/Registry-Yes-green?style=for-the-badge)
![badge](https://img.shields.io/badge/File-No-red?style=for-the-badge)
![badge](https://img.shields.io/badge/AD%2FGPO-No-red?style=for-the-badge)
![badge](https://img.shields.io/badge/AAD%2FGPO-No-red?style=for-the-badge)
![badge](https://img.shields.io/badge/-Unknown%2FUntested-lightgray?style=for-the-badge)
![badge](https://img.shields.io/badge/-Full%20Fidelity-brightgreen?style=for-the-badge)
![badge](https://img.shields.io/badge/-Unknown%2FUntested-lightgray?style=for-the-badge)
![badge](https://img.shields.io/badge/-Mostly%20Works-yellow?style=for-the-badge)
![badge](https://img.shields.io/badge/-High%20Confidence-green?style=for-the-badge)

You can delete the entire branch if no good.

**Comment by TimMangan** [comment]
Thank you for that!  I've made so many other changes, I'll just integrate directly into the main branch.  I'll remove your branch when done.
